test/integration/kubernetes_deploy_test.rb

```diff
@@ -1,27 +1,118 @@
 require 'test_helper'
 
 class KubernetesDeployTest < KubernetesDeploy::IntegrationTest
-  def test_basic
-    runner = KubernetesDeploy::Runner.new(
-      namespace: @namespace,
-      current_sha: "abcabcabc",
-      context: MINIKUBE_CONTEXT,
-      template_dir: File.expand_path("./test/fixtures/basic"),
-      wait_for_completion: true,
-    )
-    runner.run
-
-    pods = kubeclient.get_pods(namespace: @namespace)
-    managed_pods, unmanaged_pods = pods.partition do |pod|
-      pod.metadata.ownerReferences && pod.metadata.ownerReferences.first.kind == "ReplicaSet"
+  def test_full_basic_set_deploy_succeeds
+    deploy_fixture_set("basic")
+    basic = FixtureSetAssertions::Basic.new(@namespace)
```

Comment:
It'd have to be `assert_all_up('basic', @namespace)` or `assert_all_basic_up(@namespace)`, unless we consider it unlikely there'll be multiple fixture sets (hard to say--there's a second now, but we don't need any assertions for it so far). In any case, I actually started out with the latter and moved to this class-based structure as an experiment, and kept it because I found it cleaner and liked the more forceful encapsulation of the knowledge of what the fixtures contain. I'm open to switching back if you guys hate this. 😄